go/vt/servenv/buildinfo.go

```diff
@@ -33,6 +33,7 @@ var (
 	buildTime             = ""
 	buildGitRev           = ""
 	buildGitBranch        = ""
+	buildVersion          *stats.String
```

Comment:
Maybe call this `statBuildVersion` to distinguish from the other variables, which are not stats.

Reply:
https://github.com/vitessio/vitess/pull/14713/commits/cbceb850f8947ade7a84047ec56a8ac7e5375bf0